src/Npgsql/KerberosUsernameProvider.cs

```diff
@@ -47,37 +47,37 @@ static void DetectUsername()
         var process = Process.Start(processStartInfo);
         if (process is null)
         {
-            Log.Debug($"klist process could not be started");
+            Logger.LogDebug("klist process could not be started");
             return;
         }
 
         process.WaitForExit();
         if (process.ExitCode != 0)
         {
-            Log.Debug($"klist exited with code {process.ExitCode}: {process.StandardError.ReadToEnd()}");
+            Logger.LogDebug($"klist exited with code {process.ExitCode}: {process.StandardError.ReadToEnd()}");
```

Comment:
You missed several strings using interpolating formatting. Was that intentional?

Reply:
You're right - opened #4277 to track cleaning this up.